src/main/java/racinggame/domain/RacingGame.java

```diff
@@ -0,0 +1,23 @@
+package racinggame.domain;
+
+import racinggame.dto.History;
+import racinggame.random.CapacityGenerator;
+
+public class RacingGame {
+
+    private Cars cars;
+    private Integer repeatCount;
+
+    public RacingGame(int repeatCount,int carCount) {
+        this.repeatCount=repeatCount;
+        this.cars = Cars.from(carCount);
+    }
+
+    public History start(CapacityGenerator capacityGenerator) {
+        for(int i=0;i<repeatCount;i++){
```

Comment:
```suggestion
        for (int i=0; i < repeatCount; i++) {
```

불편하지만 요구사항 중 하나이니 지켜보면 좋겠습니다!

> 자바 코드 컨벤션을 지키면서 프로그래밍한다.
이 과정의 Code Style은 [intellij idea Code Style. Java](https://www.jetbrains.com/help/idea/code-style-java.html)을 따른다.
[intellij idea Code Style. Java](https://www.jetbrains.com/help/idea/code-style-java.html)을 따르려면 code formatting 단축키(Windows : Ctrl + Alt + L. Mac : ⌥ (Option) + ⌘ (Command) + L.)를 사용